mock/api.js

```diff
@@ -1,381 +1,297 @@
-              measurement_type: 'test_measurement_type',
-              measurement_idx: 0,
-              measurement_title: 'test_measurement_title',
-              '@idx': 0,
-              name: 'sample1',
-            },
-          },
-        },
-        {
-          _source: {
-            run: {
-              id: 'test_run_id',
-              controller: 'test_controller',
-              name: 'test_run_name',
-              script: 'test_script',
-              config: 'test_config',
-            },
-            iteration: { name: 'test_iteration_2', number: 2 },
-            benchmark: {
-              instances: 1,
-              max_stddevpct: 1,
-              message_size_bytes: 1,
-              primary_metric: 'test_measurement_title',
-              test_type: 'stream',
-            },
-            sample: {
-              closest_sample: 1,
-              mean: 0.1,
-              stddev: 0.1,
-              stddevpct: 1,
-              uid: 'test_measurement_id',
-              measurement_type: 'test_measurement_type',
-              measurement_idx: 0,
-              measurement_title: 'diff_measurement_title',
-              '@idx': 1,
-              name: 'sample2',
-            },
-          },
-        },
-      ],
-    },
-    aggregations: {
-      id: {
-        buckets: [
-          {
-            key: 'test_run_name',
-            type: {
-              buckets: [
-                {
-                  key: 'test_measurement_type',
-                  title: {
-                    buckets: [
-                      {
-                        key: 'test_measurement_title',
-                        uid: {
-                          buckets: [
-                            {
-                              key: 'test_measurement_id',
-                            },
-                          ],
-                        },
-                      },
-                    ],
-                  },
-                },
-              ],
-            },
-          },
+export const mockTableContents = {
+  hits: {
+    total: DEFAULT_SIZE,
+    hits: new Array(DEFAULT_SIZE).fill().map((value, index) => ({
+      _source: {
+        parent: `${casual.word}/${casual.word}/${casual.word}/${casual.word}/${casual.word}/${
+          casual.word
+        }`,
+        directory: `${casual.word}/${casual.word}/${casual.word}/${casual.word}/${casual.word}/${
+          casual.word
+        }`,
+        mtime: moment.utc() + index,
+        mode: casual.word,
+        name: casual.word,
+        ancestor_path_elements: [
+          casual.word,
+          casual.word,
+          casual.word,
+          casual.word,
+          casual.word,
+          casual.word,
         ],
-      },
-      name: {
-        buckets: [
+        files: [
           {
-            key: 'test_run_name',
+            name: casual.uid,
+            mtime: moment.utc() + index,
+            size: casual.integer(1, DEFAULT_SIZE),
+            mode: casual.word,
```

Comment:
Didn't Anisha have some outstanding code to _parse_ the mode string (once the effect actually returns it)? Is it worth building a better fake (octal numeric) mode now (or just hardcoding something like `"0o644"`?) vs deferring that to later?

Reply:
I'm certainly open to including a hardcoded value or creating a small helper to generate fake modes. @anishaswain Can you confirm if you're working on parsing the mode string as part of your work on table of contents data?

Reply:
I've updated the `mode` field here to be a hardcoded value for "0o644".